modules/install/Classes/Import.php

```diff
@@ -229,7 +233,7 @@ public function ImportXML($rewrite = null)
                                             // Handle structure changes in general
                                         } elseif ($db_field["Type"] != $type
                                         or (!($db_field["Null"] == $null or ($db_field["Null"] == 'YES' and $null == 'NULL')))
-                                        or ($db_field["Default"] != $default_xml and !($db_field["Default"] == 0 and $default_xml == '') and !($db_field["Default"] == '' and $default_xml == 0))
+                                        or ($db_field["Default"] != $default_xml and !($db_field["Default"] == 0 and $default_xml == '') and !($db_field["Default"] == '' and $default_xml === 0))
```

Comment:
Will this ever work without casting/double checks?

`$default_xml` is set by 
```php
$default_xml = $this->xml->getFirstTagContent("default", $field);
```

`getFirstTagContent` will always return a string, even is `0` is in it. 

Maybe it is better to run something like `$default_xml === '0'`?

See https://github.com/lansuite/lansuite/blob/7e16dd2e209eae33a52e4dc12f4736d5b98c2df2/inc/Classes/XML.php#L16-L41